src/Service/S3/src/Input/DeleteObjectsRequest.php

```diff
@@ -128,8 +128,15 @@ public function request(): Request
         $uri['Bucket'] = $this->Bucket ?? '';
         $uriString = "/{$uri['Bucket']}?delete";
 
+        // Prepare Body
+
+        $document = new \DOMDocument('1.0', 'UTF-8');
+        $document->formatOutput = false;
+        $this->requestBody($document, $document);
```

Comment:
This looks super weird. But I see why you are doing it. Smart!

Reply:
$document is both the root node (first argument), and node creator (second argument).

I think, I can avoid the first argument. Let me try something else

Reply:
Isnt this also needed if we are doing requests to value objects' requestBody()?

Reply:
hmm no, nevermind, the name of the tag is not related to the `shape`, but the `member`.
Meaning, the same object could be inside a `<Foo>` tag then a `<Bar>` tag. The node can not be created by the object itself, it have to be created by the caller.